Makefile

```diff
@@ -115,6 +115,11 @@ e2e-tests-self-provisioned-es: prepare-e2e-tests deploy-es-operator
 	@echo Running Self provisioned Elasticsearch end-to-end tests...
 	@STORAGE_NAMESPACE=$(STORAGE_NAMESPACE) ES_OPERATOR_NAMESPACE=$(ES_OPERATOR_NAMESPACE) ES_OPERATOR_IMAGE=$(ES_OPERATOR_IMAGE) go test -tags=self_provisioned_elasticsearch ./test/e2e/... $(TEST_OPTIONS)
 
+.PHONY: e2e-tests-self-provisioned-es-kafka
+e2e-tests-self-provisioned-es-kafka: prepare-e2e-tests deploy-kafka-operator
```

Comment:
@kevinearls could you please create a jira to run this in the internal build?

Reply:
@pavolloffay  I've added a note to TRACING-806 to check once this is merged.  I think it will cover that case.